tests/tizen_sensor/unittest_tizen_capi.cc

```diff
@@ -25,6 +25,13 @@ constexpr bool is_enabled_tensorflow_lite = true;
 constexpr bool is_enabled_tensorflow_lite = false;
 #endif /* defined (ENABLE_TENSORFLOW_LITE) */
 
+
+#ifdef ENABLE_TENSORFLOW_LITE
```

Comment:
Now testcases for c-api will be removed. (see #3078).
plz update this PR without this file.

Reply:
This file has been removed from the PR. Thank you!